packages/utils/ast-utils.js

```diff
@@ -429,6 +429,62 @@ function addProperty(j, p, key, value) {
 	}
 	return;
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
+	let valForNode;
+	if (!p) {
+		return;
+	}
+	if (Array.isArray(value)) {
+		let arrExp = j.arrayExpression([]);
```

Comment:
`arrExp` can a different name. Maybe `valueForNodeAsArray`? Or something like that means something

Reply:
Yes, I get that. I am more up for names that mean something based on the context of the operation. Just that.

Reply:
Maybe a comment will do, if you follow common sense we check if the value is an array exp, so we want to create an array node